cookbook/deployment/heroku.rst

```diff
@@ -217,6 +217,84 @@ application will respond:
 
 You should be seeing your Symfony application in your browser.
 
+Custom Compile Steps
+~~~~~~~~~~~~~~~~~~~~
+
+If you wish to execute additional custom commands during a build, you can leverage
+Heroku's `custom compile steps`_. Let's say for example you want to remove the ``dev``
```

Comment:
we always try to avoid first person usage. Let's update this to "Imagine you want to remove the [...]"